src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
```

Comment:
Ensure that the element with the class 'tree' exists in the DOM before querying it. If it doesn't exist, `tree` will be `null`, and subsequent operations will fail.